aspnetcore/razor-components/index.md

```diff
@@ -1,65 +1,37 @@
 ---
-title: Introduction to Razor Components
+title: Introduction to ASP.NET Core Razor Components
 author: guardrex
-description: Explore Blazor, a .NET web framework using C#/Razor and HTML that runs in the browser with WebAssembly.
+description: Explore ASP.NET Core Razor Components, a .NET web framework using C#/Razor and HTML.
 monikerRange: '>= aspnetcore-3.0'
 ms.author: riande
 ms.custom: mvc
-ms.date: 01/29/2019
+ms.date: 02/05/2019
 uid: razor-components/index
 ---
-# Introduction to Razor Components
+# Introduction to ASP.NET Core Razor Components
 
 By [Steve Sanderson](http://blog.stevensanderson.com), [Daniel Roth](https://github.com/danroth27), and [Luke Latham](https://github.com/guardrex)
 
 [!INCLUDE[](~/includes/razor-components-preview-notice.md)]
 
-*ASP.NET Core Razor Components* executes server-side in ASP.NET Core, while *Blazor* (Razor Components executed client-side) is an experimental .NET web framework using C#/Razor and HTML that runs in the browser with WebAssembly. Blazor provides all of the benefits of a client-side web UI framework using .NET on the client.
+*ASP.NET Core Razor Components* is a .NET web framework using C#/Razor and HTML. *Blazor* is the client-side hosting model of Razor Components.
```

Comment:
This first intro paragraph needs to say what Razor Components are and what they are for.